services/common/src/components/documents/DocumentColumns.tsx

```diff
@@ -78,7 +79,7 @@ export const renderDocumentLinkColumn = (
   title = "File Name",
   sortable = true,
   docManGuidIndex = "document_manager_guid"
-): ColumnType<MineDocument> => {
+): ColumnType<MineDocument | IPermitAmendmentDocument> => {
```

Comment:
this is a _really specific_ type of document. Does it not work to have IPermitAmendmentDocument extend IMineDocument?

Reply:
It has less fields than a MineDocument and actually has it's own DB table.  For the purposes of this column they both work, but I think it's better to keep them distinct.

Reply:
bah I'll accept it, even if I don't like it